GoogleForCreators/web-stories-wp
# Use Moveable for Text Sets

## Summary
Uses Moveable for Text sets. 


## Relevant Technical Choices
Uses Moveable for text sets. Once all elements use Moveable, snapping can be implemented


## To-do

- [x] Adjust tests



## User-facing changes
None.


## Testing Instructions
1. Verify that clicking on a Text Set in the library will add the text set as expected.
2. Verify that dragging a Text Set to canvas positions the text set to the place where it was dropped.


---



Fixes #5337
